packages/components/notification/src/notification.ts

```diff
@@ -24,8 +24,7 @@ export const notificationProps = buildProps({
     default: 4500,
   },
   icon: {
-    type: definePropType<string | Comment>([String, Object]),
-    default: '',
+    type: iconPropType,
```

Comment:
Missing default.

Reply:
I think it's OK not to specify a default value, and I will fix the test case later. WDYT?

Reply:
I created a PR to refactor `IconTypes`.

Reply:
Okey.